samples/nrf9160/serial_lte_modem/src/tcpip/th_tcpip_service.c

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(tcpip, CONFIG_THIN_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <uart.h>
+#include <string.h>
+#include <net/socket.h>
+#include "th_connect.h"
+#include "th_util.h"
+
+#define TCPIP_BUFFER_SIZE	(CONFIG_INTER_CONNECT_UART_BUF_SIZE - 4)
```

Comment:
Why `- 4`?

Reply:
4 is the reserved size for protocol header/trailer